app/components/structure/contact_form_component.html.erb

```diff
@@ -0,0 +1,30 @@
+<p>* Required fields</p>
+
+<%= form_with model: ContactForm.new do |form| %>
```

Comment:
I didn't find the `@contact_form` instance variable in the controller to be necessary, so I dropped it.